app.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Whedon API",
+  "description": "The Whedon Bot for interacting with reviewers on JOSS, JOSE and other journals",
+  "keywords": [
+    "Whedon",
+    "JOSS",
+    "Open Journals",
+    "bots"
+  ],
+  "website": "http://openjournals.org",
+  "repository": "https://github.com/openjournals/whedon-api/tree/multi-whedon",
+  "success_url": "/heartbeat",
+  "scripts": {
+  },
+  "env": {
+    "JOSS_API_KEY": {
+      "description": "The API key for the JOSS site API"
+    },
+    "JOSE_API_KEY": {
+      "description": "The API key for the JOSE site API"
+    },
+    "JOSS_API_KEY_TESTING": {
+      "description": "Test API key"
+    },
+    "GH_TOKEN": {
+      "description": "The GitHub token for Whedon"
+    }
+  },
+  "formation": {
+    "web": {
+      "quantity": 1,
+      "size": "hobby"
+    },
+    "worker": {
+      "quantity": 1,
+      "size": "hobby"
+    }
+  },
+  "addons": [ "redistogo:nano", "heroku-postgresql:hobby-dev"],
+  "buildpacks": [
+    {
+      "url": "https://github.com/openjournals/heroku-buildpack-tex"
```

Comment:
This gives us the TeXLive environment we will need to build papers with the `Whedon` gem